docs/articles_en/documentation/openvino-ir-format/operation-sets/operations-specifications/pooling/avg-pool-14.rst

```diff
@@ -0,0 +1,206 @@
+.. {#openvino_docs_ops_pooling_AvgPool_14}
+
+AvgPool
+=======
+
+
+.. meta::
+  :description: Learn about AvgPool-1 - a pooling operation, which can 
+                be performed on a 3D, 4D or 5D input tensor.
+
+**Versioned name**: *AvgPool-1*
+
+**Category**: *Pooling*
+
+**Short description**: `Reference <http://caffe.berkeleyvision.org/tutorial/layers/pooling.html>`__
```

Comment:
Let us have our own short description for this operation. The link to Caffe looks not cool.

Reply:
Good point, I replaced it.